apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -97,3 +119,15 @@ const loadAssessmentsFromServer = (sectionId, scriptId, onComplete) => {
     onComplete(status, jqXhr.responseJSON);
   });
 };
+
+const loadAssessmentsStructureFromServer = (scriptId, onComplete) => {
+  $.ajax({
+    url: `/dashboardapi/assessments_structure/${scriptId}`,
+    method: 'GET',
+    contentType: 'application/json;charset=UTF-8',
+  }).done(assessmentsData => {
+    onComplete(null, assessmentsData);
+  }).fail((jqXhr, status) => {
+    onComplete(status, jqXhr.responseJSON);
+  });
```

Comment:
you could return the result of `$.ajax` from this function, and then call `then` and `catch` on it